workflow-testing/src/main/java/com/squareup/workflow/testing/RenderTestResult.kt

```diff
@@ -49,5 +49,26 @@ interface RenderTestResult<StateT, OutputT : Any> {
    * be useful if your actions anonymous. If they are a sealed class or enum, use [verifyAction]
    * instead and write separate unit tests for your action implementations.
    */
-  fun verifyActionResult(block: (newState: StateT, output: OutputT?) -> Unit)
+  fun verifyActionResult(
+    block: (
+      newState: StateT,
+      hasOutput: Boolean,
+      output: OutputT
+    ) -> Unit
+  )
+}
+
+// TODO Should this overload be provided? Would be even better if we could find some way to get
```

Comment:
Ticket?

Reply:
No, I want to figure it out before merging.